tidy3d/components/geometry.py

```diff
@@ -1206,6 +1321,75 @@ def bounds(self) -> Bound:
         coord_max[self.axis] = self.center[self.axis] + self.length / 2.0
         return (tuple(coord_min), tuple(coord_max))
 
+    def volume(self, bounds: Bound) -> float:
+        """Returns object's volume. If the object extends beyound bounds then
+        the computed volume may be only a rough approximation
+
+        Returns
+        -------
+        float
+            Volume.
+        """
+
+        coord_min = self.center[self.axis] - self.length / 2.0
```

Comment:
I think `coord_min` and `coord_max` are just `self.bounds[0][self.axis]` and  `self.bounds[1][self.axis]` respectively?